### ISSUE TITLE
:help command lists command aliases with the name of the original command

### ISSUE THREAD

**Issue by ryanrousseau** [scriptcs/scriptcs#1106]
If I add an alias for :clear called :cls and then run the :help command, "clear" will show up twice in the list.

![image](https://cloud.githubusercontent.com/assets/1571799/10652715/7c838a48-781e-11e5-884c-8544e2433939.png)

It would be nice to show the the command alias in the help list as well.

How would you feel about adding a new class AliasedCommand (I'm not a fan of the name because it's too close to AliasCommand, but I haven't come up with anything better yet)

```csharp
    public class AliasedCommand : IReplCommand
    {
        private readonly string _Alias;
        private readonly IReplCommand _Command;

        public string Description
        {
            get
            {
                return _Command.Description;
            }
        }

        public string CommandName
        {
            get
            {
                return _Alias;
            }
        }

        public object Execute(IRepl repl, object[] args)
        {
            return _Command.Execute(repl, args);
        }

        public AliasedCommand(string alias, IReplCommand command)
        {
            _Alias = alias;
            _Command = command;
        }
    }
```

and replacing

```csharp
            repl.Commands[alias] = command;
```

with

```csharp
            repl.Commands[alias] = new AliasedCommand(alias, command);
```

**Comment by ryanrousseau** [comment]
Found this was fixed in 0.15.0, I had svm set to 0.14.1 for testing something else.

**Comment by adamralph** [comment]
Dup of https://github.com/scriptcs/scriptcs/issues/985
